magento/magento-cloud-patches
# BUNDLE-2683: VBE Braintree - Error occurs in Admin Braintree Settlement Report

### Description:
https://jira.corp.magento.com/browse/MCLOUD-6684:
Adds patch to fix issue for Braintree VBE where Admin viewing Braintree Settlement Report causes error screen

### Fixed Issues:
https://jira.corp.magento.com/browse/BUNDLE-2683

### Manual Testing Scenarios
1. Enable Braintree
2. Navigate to: Magento Admin Panel > Reports > Braintree Settlement
3. “Braintree Settlement Report” page should be loaded
